spicy/runtime/src/sink.cc

```diff
@@ -1,5 +1,7 @@
 // Copyright (c) 2020-2023 by the Zeek Project. See LICENSE for details.
 
+#include "spicy/rt/sink.h"
```

Comment:
#include <spicy/rt/sink.h>

Reply:
Even though we still do not do this consistently, I would really like to include this with `#include "..."` to make clear that this is [the header related to this .cc file](https://google.github.io/styleguide/cppguide.html#Names_and_Order_of_Includes). With that we e.g., get it to sort before other headers which helps make sure that the header is self-contained.

Reply:
I think we had this discussion a few times already. :-) I won't fight it, I just don't like the inconsistency of doing it randomly for a few implementation files, but not most others.